<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ImageAcquisitionWidget</class>
 <widget class="QWidget" name="ImageAcquisitionWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1047</width>
    <height>706</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ImageAcquisitionWidget</string>
  </property>
  <layout class="QGridLayout" name="gridLayout" columnstretch="90,10" columnminimumwidth="90,10">
   <item row="0" column="0">
    <widget class="QWidget" name="ultrasoundDisplay" native="true"/>
   </item>
   <item row="0" column="1">
    <layout class="QVBoxLayout" name="verticalLayout_2" stretch="20,0,40">
     <item>
      <widget class="QWidget" name="imagesDisplay" native="true"/>
     </item>
     <item>
      <widget class="QSlider" name="imageSlider">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="5" column="0">
        <widget class="QLineEdit" name="coordsY"/>
       </item>
       <item row="6" column="0">
        <widget class="QLineEdit" name="coordsZ"/>
       </item>
       <item row="20" column="1">
        <widget class="QPushButton" name="quitBtn">
         <property name="text">
          <string>Quit</string>
         </property>
        </widget>
       </item>
       <item row="5" column="1">
        <widget class="QLineEdit" name="rotX"/>
       </item>
       <item row="6" column="1">
        <widget class="QLineEdit" name="rotY"/>
       </item>
       <item row="7" column="1">
        <widget class="QLineEdit" name="rotZ"/>
       </item>
       <item row="19" column="1">
        <widget class="QPushButton" name="saveImagesBtn">
         <property name="text">
          <string>Save Images</string>
         </property>
        </widget>
       </item>
       <item row="18" column="1">
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Tanslation</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Rotation</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLineEdit" name="coordsX"/>
       </item>
       <item row="2" column="1">
        <widget class="QLineEdit" name="rotW"/>
       </item>
       <item row="8" column="1">
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="13" column="1">
        <widget class="QLineEdit" name="numberImages"/>
       </item>
       <item row="13" column="0">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Number of Images</string>
         </property>
        </widget>
       </item>
       <item row="14" column="0">
        <widget class="QPushButton" name="singleImageBtn">
         <property name="text">
          <string>Single Image</string>
         </property>
         <property name="checkable">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item row="14" column="1">
        <widget class="QPushButton" name="multipleImageBtn">
         <property name="text">
          <string>Multiple Images</string>
         </property>
         <property name="checkable">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item row="9" column="0">
        <widget class="QPushButton" name="configTrackerBtn">
         <property name="text">
          <string>Config Tracker</string>
         </property>
        </widget>
       </item>
       <item row="9" column="1">
        <widget class="QPushButton" name="startTrackingBtn">
         <property name="text">
          <string>Start Tracking</string>
         </property>
        </widget>
       </item>
       <item row="10" column="0">
        <spacer name="verticalSpacer_3">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="15" column="1">
        <widget class="QLabel" name="trackingLabel">
         <property name="text">
          <string>Acquiring Images</string>
         </property>
        </widget>
       </item>
       <item row="15" column="0">
        <widget class="QPushButton" name="clearBtn">
         <property name="text">
          <string>Clear Images</string>
         </property>
        </widget>
       </item>
       <item row="11" column="0">
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>Wait to acquire</string>
         </property>
        </widget>
       </item>
       <item row="11" column="1">
        <widget class="QLineEdit" name="waitSeconds"/>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>singleImageBtn</sender>
   <signal>clicked()</signal>
   <receiver>ImageAcquisitionWidget</receiver>
   <slot>singleImage()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>927</x>
     <y>555</y>
    </hint>
    <hint type="destinationlabel">
     <x>1042</x>
     <y>286</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>multipleImageBtn</sender>
   <signal>clicked()</signal>
   <receiver>ImageAcquisitionWidget</receiver>
   <slot>multipleImages()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>1033</x>
     <y>555</y>
    </hint>
    <hint type="destinationlabel">
     <x>1039</x>
     <y>319</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>configTrackerBtn</sender>
   <signal>clicked()</signal>
   <receiver>ImageAcquisitionWidget</receiver>
   <slot>configTracker()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>927</x>
     <y>460</y>
    </hint>
    <hint type="destinationlabel">
     <x>1041</x>
     <y>476</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>startTrackingBtn</sender>
   <signal>clicked()</signal>
   <receiver>ImageAcquisitionWidget</receiver>
   <slot>startTracking()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>1033</x>
     <y>460</y>
    </hint>
    <hint type="destinationlabel">
     <x>1044</x>
     <y>531</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>quitBtn</sender>
   <signal>clicked()</signal>
   <receiver>ImageAcquisitionWidget</receiver>
   <slot>Quit()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>978</x>
     <y>673</y>
    </hint>
    <hint type="destinationlabel">
     <x>1040</x>
     <y>654</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>saveImagesBtn</sender>
   <signal>clicked()</signal>
   <receiver>ImageAcquisitionWidget</receiver>
   <slot>saveImages()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>987</x>
     <y>640</y>
    </hint>
    <hint type="destinationlabel">
     <x>1045</x>
     <y>623</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>imageSlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>ImageAcquisitionWidget</receiver>
   <slot>displaySelectedImage(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>863</x>
     <y>244</y>
    </hint>
    <hint type="destinationlabel">
     <x>814</x>
     <y>359</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>clearBtn</sender>
   <signal>clicked()</signal>
   <receiver>ImageAcquisitionWidget</receiver>
   <slot>clearImages()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>872</x>
     <y>574</y>
    </hint>
    <hint type="destinationlabel">
     <x>815</x>
     <y>541</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>singleImage()</slot>
  <slot>multipleImages()</slot>
  <slot>configTracker()</slot>
  <slot>startTracking()</slot>
  <slot>Quit()</slot>
  <slot>saveImages()</slot>
  <slot>displaySelectedImage(int)</slot>
  <slot>clearImages()</slot>
 </slots>
</ui>
